### DISCUSSION TITLE
[Feature Request] Option to disable "...is typing" notification

### DISCUSSION THREAD

**Discussion by NMBlanco** [rodit/SnapMod#134] [Ideas & Feature Suggestions]
Back in SnapTools, we could disable the annoying notification "... is typing" 
It would be cool if you are able to add that option!

**Comment by rodit** [comment]
There is an option to filter notifications in 1.7.0 (just released) - filter 'Typing' notifications (I haven't tested that particular filter but the others work correctly).

**Reply by RevealedSoulEven** [reply]
I already told that in one of my discussion. Which is released yet
